.github/workflows/ci.yml

```diff
@@ -57,12 +57,9 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-clippy-${{ hashFiles('**/Cargo.lock') }}
       - name: Select Toolchain
-        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
+        uses: dtolnay/rust-toolchain@7b1c307e0dcbda6122208f10795a713336a9b35a
```

Comment:
Would you mind adding the "human" version next to the shasum?

Reply:
There is none unfortunately, I opened https://github.com/dtolnay/rust-toolchain/issues/121 because of that.

Reply:
I've seen the response on the linked issue. There's nothing we can do about that